### PR TITLE
Feature/pdf response

### PR DESCRIPTION
**Pull request by larasmorningtrain**

### CODE REVIEW

File: src/DTOs/Invoice/Pdf.php [hunk 1]
```diff
@@ -3,8 +3,17 @@
...
 
 class Pdf extends Resource
 {
     public ?string $download;
+
+    public function getPdfContentResponse(): EconomicResponse
+    {
+        $url = $this->download;
+
+        return EconomicApiService::get($url);
```
Comment (mschadegg) [inline src/DTOs/Invoice/Pdf.php:17]: There are no reason to set url before using it.

File: src/DTOs/Invoice/Pdf.php [hunk 2]
```diff
@@ -3,8 +3,17 @@
...
 
 use Morningtrain\Economic\Abstracts\Resource;
+use Morningtrain\Economic\Classes\EconomicResponse;
+use Morningtrain\Economic\Services\EconomicApiService;
 
 class Pdf extends Resource
 {
     public ?string $download;
+
+    public function getPdfContentResponse(): EconomicResponse
```
Comment (mschadegg) [inline src/DTOs/Invoice/Pdf.php:13]: Return only the body, and rename to getPdfContent

**Review by mschadegg — CHANGES_REQUESTED** [review]

**Comment by mschadegg** [comment]
Please add some short documentation on how to use it in the README file, and maybe as PHP DOC BLOCK on the function.

File: src/Classes/EconomicResponse.php [hunk 3]
```diff
@@ -6,9 +6,9 @@ class EconomicResponse
 {
     protected int $statusCode;
 
-    protected array $body;
+    protected mixed $body;
 
-    public function __construct(int $statusCode, array $body)
+    public function __construct(int $statusCode, mixed $body)
```
Comment (SimonJnsson) [inline src/Classes/EconomicResponse.php:11]: Can we type hint more specifically than "mixed"?

**Comment by mschadegg** [comment]
Ohh @larasmorningtrain  You also have to typehint it correctly in the constructor and the getBody.

**Pull request merged**